catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -511,6 +534,24 @@ private QueryRequest validateQueryRequest(QueryRequest queryRequest)
                     "Cannot perform query with null query, either passed in from endpoint, or as output from a PreQuery Plugin");
         }
 
+        Query originalQuery = queryRequest.getQuery();
```

Comment:
Not all that much cleaner IMO, but I like not re-assigning `queryRequest`. Fixed.